### DISCUSSION TITLE
Is the download counter limited by something?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3125] [Q&A Feedback] [answered]
Within the readme, the project displays a badge with the amount of downloads of Sandboxie.

Is the displayed amount over all releases, or just over the last x-releases or over a given timeframe?

I ask this question since the displayed value, was already at that level, some time ago, but then the displayed value was lower.

**Comment by DavidXanatos** [comment, marked as answer]
It seams it only displays the last x releases,
as you can see here: https://tooomm.github.io/github-release-stats/?username=DavidXanatos&repository=Sandboxie
every major release has around 100.000 downloads
so we should be at many many many milions by now.

we probably should remove that badge as its not correct.

**Reply by bastik-1001** [reply]
The statistics page you linked, sums up to 1.6 millions, which is even less, probably as it does not list all releases. It is still interesting, though, since there are lots of 32bit downloads. More than I had expected. I already that the classic version is still pretty popular.

**Reply by isaak654** [reply]
> as you can see here: https://tooomm.github.io/github-release-stats/?username=DavidXanatos&repository=Sandboxie
> every major release has around 100.000 downloads

Not really, according to that link:
- [v1.8.0](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.8.0) has 7.564 downloads;
- [v1.9.0](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.0) has 4.472 downloads;
and so on.

**Reply by offhub** [reply]
They were pre-release versions.

**Reply by DavidXanatos** [reply]
[1.9.8](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.8) has 76.873
[v1.9.6](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.6) has 191.758
and [v1.7.2](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.7.2) cracked the quatre million with 286.537 downloads

**Reply by isaak654** [reply]
> They were pre-release versions.

It's also true that sometimes a major release could also be a pre-release, therefore defining a release as major can be misunderstood.

**Reply by bastik-1001** [reply]
> It's also true that sometimes a major release could also be a pre-release, therefore defining a release as major can be misunderstood.

Sure, 1.10.0 (was also a pre-release and so is 1.10.1) where one could believe that it would be a normal release, just by looking at the version number, with some big changes. Well, it contains big changes, just the release is not released as stable. This appears to be different from other software, like browsers, but those tend to have a different approach to version numbers, too.

**Reply by DavidXanatos** [reply]
Well ignoreing the past past the plan is x.yy.zzz
x must be a huge change extremely huge
yy somethign is new not just bug fixes
zzz every change released on github
every new 1.xx starts as a prerelease untill we are sure that all works fine than it advances to a full release with zzz at whatever count of iterations it took to arive there

there can also be x.yy.zzzL where L is a-z this releases are only pushed through the online updater and are unsigned.

**Reply by bastik-1001** [reply]
You could, even with the semantic versioning system, do x.yy.zzz[-string]

Like one of these:
1.11.0-Alpha1
1.11.0-Alpha2
1.11.0-Alpha3
1.11.0-Beta1
1.11.0-Beta2
1.11.0-Beta3
1.11.0-Beta4
1.11.0-RC1 (or ReleaseCandidate1, or pre-release1)
1.11.0-RC2
1.11.0

where the last is the final release, without any suffix.

Update would be:
1.11.1
1.11.2

following the same logic as above when needed:
1.11.3-Beta1 comes before
1.11.3
